pypesto/optimize/ess/ess.py

```diff
@@ -172,6 +172,7 @@ def _initialize(self):
         # Index of current iteration
         self.n_iter: int = 0
         # ESS iteration at which the last local search took place
+        # (only local searches with a finite result are counted)
```

Comment:
That's a good addition here, I assume (looking at the code) that the problem occurred since neither local solution was set nor did self.n_iter go up? In this case a similar info on `self.local_solutions` might also be good?